addons/website/models/website_form.py

```diff
@@ -60,7 +60,7 @@ def get_authorized_fields(self, model_name):
         # Remove readonly and magic fields
         # Remove string domains which are supposed to be evaluated
         # (e.g. "[('product_id', '=', product_id)]")
-        MAGIC_FIELDS = models.MAGIC_COLUMNS + [model.CONCURRENCY_CHECK_FIELD]
+        MAGIC_FIELDS = models.MAGIC_COLUMNS
```

Comment:
Nitpicking
We could save a variable and remplace all usages of `MAGIC_FIELDS` by `models.MAGIC_COLUMNS`